angularSTEP/src/app/recipe-page/recipe-page.component.scss

```diff
@@ -0,0 +1,38 @@
+.solo-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-top: 2%;
+}
+
+.solo-item {
+  margin-bottom: .5%;
+  width: 33%
+}
+
+.item-button-pair {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: 50%;
```

Comment:
FYI using exclusively % values for layout generally isn't very good for supporting extremely large/small screens

Reply:
I'll keep that in mind! I think when I was adding the styling I was trying to have the list of items be centered on the page while still being aligned at the start with each other and the resulting code is what got me closest to the layout I had imagined. Admittedly I am not the best at using CSS so there are probably techniques that I'm unaware of that would have helped me achieve the same end result.